squad/core/models.py

```diff
@@ -979,6 +979,42 @@ def full_name(self):
             suite = self.metadata.suite
         return join_name(suite, self.name)
 
+    class Confidence(object):
+        def __init__(self, passes, count, threshold):
+            self.passes = passes
+            self.count = count
+            self.threshold = threshold
```

Comment:
`threshold` does not seem to be used for anything.

Reply:
Fixed, it's now printed in the tooltip, with more use intended later on.